pkg/webhook/v1alpha2/applicationconfiguration/applicationconfiguration.go

```diff
@@ -27,6 +36,8 @@ var appConfigResource = v1alpha2.SchemeGroupVersion.WithResource("applicationcon
 type ValidatingHandler struct {
 	Client client.Client
 
+	AppsClient clientappv1.AppsV1Interface
```

Comment:
I wonder if we could use `Client client.Client` instead of create a new one?

Reply:
Yes, that would be more concise, but render.go/getComponent [use clientappv1 to get controllerrevision](https://github.com/crossplane/oam-kubernetes-runtime/blob/ca3bc22d767e0a67da3eeb8bc07360916f544d8d/pkg/controller/v1alpha2/applicationconfiguration/render.go#L254) and I need to reuse that func. So that may be trade-off...

Reply:
Can you refactor that function?

Reply:
Yes, I will fix it.

Reply:
> Could you help refactor that function?

I create an issue #152 for this and will raise another PR to fix it.

Reply:
cool, thanks!